modules/sharedaddy/sharing.js

```diff
@@ -21,7 +21,7 @@ if ( sharing_js_options && sharing_js_options.counts ) {
 				requests = {
 					// LinkedIn actually gets the share count for both the http and https version automatically -- so we don't need to do extra magic
 					linkedin: [
-							'https://www.linkedin.com/countserv/count/share?format=jsonp&callback=updateLinkedInCount&url=' +
+						'https://www.linkedin.com/countserv/count/share?format=jsonp&callback=WPCOMSharing.update_linkedin_count&url=' +
```

Comment:
I'm slightly confused why it wasn't by the automated system -- I'm going back to the wpcom source.  My goal right now is to just get the two files in sync, and we can do minor code tidying in subsequent changes.